Dockerfile.gnoland.release

```diff
@@ -1,13 +0,0 @@
-FROM busybox
```

Comment:
I missed that we now had five different Dockerfiles. Thank you for merging them.